src/pybind/mgr/nfs/module.py

```diff
@@ -70,14 +70,11 @@ def _cmd_nfs_cluster_create(self,
                                 ingress: Optional[bool]=None,
                                 virtual_ip: Optional[str]=None) -> Tuple[int, str, str]:
         """Create an NFS Cluster"""
-        if virtual_ip and not ingress:
-            return (-errno.EINVAL, '',
-                    '--virtual-ip can only be provided with --ingress')
-        if ingress and not virtual_ip:
-            return (-errno.EINVAL, '',
-                    '--ingress current requires --virtual-ip')
-        return self.nfs.create_nfs_cluster(cluster_id=clusterid, placement=placement,
-                                           virtual_ip=virtual_ip)
+        try:
+            return self.nfs.create_nfs_cluster(cluster_id=clusterid, placement=placement,
+                                               virtual_ip=virtual_ip, ingress=ingress)
+        except orchestrator.OrchestratorError as e:
+            return (-errno.EINVAL, '', str(e))
```

Comment:
Here `try-except` block is not required, if above suggested changes are made

Reply:
Please update doc about virtual ip and ingress https://ceph--41007.org.readthedocs.build/en/41007/cephfs/fs-nfs-exports/